JSDemos/Demos/SelectBox/GroupedItems/description.md

```diff
@@ -18,6 +18,18 @@ The SelectBox can also work with initially grouped data items. In this case, the
         ]
     }];
 
-If data objects are grouped but use other field names, implement the **DataSource**'s [map](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#map) function to create **key** + **items** field mappings.
+If data objects are grouped but use other field names, implement the **DataSource**'s [map](/Documentation/ApiReference/Data_Layer/DataSource/Configuration/#map) function to create **key** + **items** field mappings as in this demo's second SelectBox.
 
-[note]Subgroups are not supported.
\ No newline at end of file
+Regardless of the data source structure, enable the [grouped](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#grouped) property.
+
+If the data source contains objects, specify the following SelectBox properties:
+
+* [valueExpr](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#valueExpr)            
+A data field that contains unique values used to identify items.
+
+* [displayExpr](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#displayExpr)             
+A data field whose values should be displayed in the drop-down list.
+
+If you need to specify a custom template for group captions, use the [groupTemplate](/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#groupTemplate) property. In this demo, each caption contains an icon and a text.
+
+[note]Only one-level grouping is supported.
```

Comment:
Place the note before the "Regardless of ..." sentence because that where we finished talking about grouping the data objects.